test/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ subdirs(
         table_printer
         network
         process
-        vega_renderer
+        # vega_renderer
```

Comment:
I will clean this up before merge.

Reply:
Please simply do not push local debugging changes to pull requests. It's too error-prone to ask reviewers to ignore certain changes and assume that we'll remember not to include them. Just have these in a separate commit locally that you can stash/cherry-pick/whatever

Reply:
That's fair. I will keep this local.